playbooks/aws/openshift-cluster/terminate.yml

```diff
@@ -40,7 +40,7 @@
         host-type: "{{ hostvars[item]['ec2_tag_host-type'] }}"
         sub_host_type: "{{ hostvars[item]['ec2_tag_sub-host-type'] }}"
     with_items: "{{ groups.oo_hosts_to_terminate }}"
-    when: "'oo_hosts_to_terminate' in groups"
+    when: 'oo_hosts_to_terminate' in groups
```

Comment:
Fixed in https://github.com/openshift/openshift-ansible/pull/4527/commits/4f04379fa47bc451e145d86197179bf240a085f2